.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
```

Comment:
_Security Check and Infrastructure Planning_ ?

Reply:
Como se trata de um build de CI (fazer validações antes de deixar qualquer coisa passar pra `master`), acho que não faz muito sentido descrever o que tem dentro. Até porque pode mudar no futuro, conforme formos adicionando mais checks e comportamentos. "CI" é breve e vago, mas era isso que estava buscando, mesmo. O que acha?